static/src/javascripts/boot.js

```diff
@@ -73,161 +72,166 @@ const go = () => {
-            } else {
-                window.addEventListener('load', captureOphanInfo);
-            }
-        });
-    });
+			if (!consentState) return;
+
+			const decideConsentCarrierLabels = () => {
+				if (consentState.tcfv2) {
+					const consentUUID = getCookie('consentUUID') || '';
+					const consentString = consentState.tcfv2?.tcString;
+					return [
+						'01:TCF.v2',
+						`02:${consentUUID}`,
+						`03:${consentString}`,
+					];
+				}
+				if (consentState.ccpa) {
+					const ccpaUUID = getCookie('ccpaUUID') || '';
+					const flag = consentState.ccpa?.doNotSell
+						? 'true'
+						: 'false';
+					return ['01:CCPA', `04:${ccpaUUID}`, `05:${flag}`];
+				}
+				if (consentState.aus) {
+					const ccpaUUID = getCookie('ccpaUUID') || '';
+					const consentStatus = getCookie('consentStatus') || '';
+					const personalisedAdvertising = consentState.aus
+						?.personalisedAdvertising
+						? 'true'
+						: 'false';
+					return [
+						'01:AUS',
+						`06:${ccpaUUID}`,
+						`07:${consentStatus}`,
+						`08:${personalisedAdvertising}`,
+					];
+				}
+				return [];
+			};
+
+			const componentType = 'CONSENT';
+
+			const action = 'MANAGE_CONSENT';
+
+			const event = {
+				component: {
+					componentType,
+					products: [],
+					labels: decideConsentCarrierLabels(),
+				},
+				action,
+			};
+
+			ophan.record({
+				componentEvent: event,
+			});
+
+			// ------------------------------------------------------
+		});
+
+		cmp.init({ pubData, country: await getLocale() });
+
+		detectAdBlockers();
+
+		// 2. once standard is done, next is commercial
+		// Handle ad blockers
+		window.guardian.adBlockers.onDetect.push((adblockInUse) => {
+			if (!adblockInUse) return;
+
+			// For the moment we'll hide the top-above-nav slot if we detect that the user has ad blockers enabled
+			// in order to avoid showing them a large blank space.
+			// TODO improve shady pie to make better use of the slot.
+			document.querySelector('.top-banner-ad-container').style.display =
+				'none';
+
+			if (process.env.NODE_ENV !== 'production') {
+				const needsMessage =
+					adblockInUse && window.console && window.console.warn;
+				const message =
+					'Do you have an adblocker enabled? Commercial features might fail to run, or throw exceptions.';
+				if (needsMessage) {
+					window.console.warn(message);
+				}
+			}
+		});
+
+		const fakeBootCommercial = { bootCommercial: () => {} };
+		const commercialBundle = () =>
+			config.get('switches.standaloneCommercialBundle') &&
+			!config.get('page.isHosted', false)
+				? loadScript(config.get('page.commercialBundleUrl')).then(
+						() => fakeBootCommercial,
+				  )
+				: import(
+						/* webpackChunkName: "commercial" */
+						'bootstraps/commercial-legacy'
+				  );
+
+		// Start downloading these ASAP
+
+		// eslint-disable-next-line no-nested-ternary
+		const fetchCommercial =
+			config.get('switches.commercial') &&
+			!config.get('page.isAdFree', false)
```

Comment:
Even with the whitespace diffing, I struggled to find this line, which is the only real meaningful change.

Could you split the formatting commit into its own? Do we need to be formatting this file?

Reply:
oh sorry, i didn't notice that had happened! how can that happen? but no it does not, will remove it

Reply:
prettier is installed but not run, apologies i didn't notice

Reply:
removed it